common/src/main/java/com/wynntils/core/consumers/features/FeatureManager.java

```diff
@@ -452,6 +454,12 @@ public void disableFeature(Feature feature) {
         Managers.Overlay.disableOverlays(feature);
 
         Managers.KeyBind.disableFeatureKeyBinds(feature);
+
+        if (McUtils.mc().screen instanceof WynntilsBookSettingsScreen settingsScreen) {
```

Comment:
This logic does not belong here. You can tell the screen that it might need to do an update, like this:
`settingScreen.onFeatureUpdated(feature)`
or something like that. And then the screen will need to make sure the button is in the correct state.